composer.json

```diff
@@ -27,7 +27,8 @@
         "ext-curl": "*"
     },
     "require-dev": {
-        "satooshi/php-coveralls": "dev-master"
+        "satooshi/php-coveralls": "dev-master",
+        "phpunit/phpunit": ">=4.0.0"
```

Comment:
well, I personally do not have phpunit installed globally, but tend to install it inside every project I develop. This because there have been in the past incompatible updates, so different projects were stuck with different phpunit versions.
But I can move this to a separate PR if you think